pep_sphinx_extensions/pep_theme/templates/page.html

```diff
@@ -10,14 +10,18 @@
     <link rel="canonical" href="{{ pageurl|escape }}" />
     <link rel="stylesheet" href="{{ pathto('_static/style.css', resource=True) }}" type="text/css" />
     <link rel="stylesheet" href="{{ pathto('_static/mq.css', resource=True) }}" type="text/css" />
-    <link rel="stylesheet" href="{{ pathto('_static/dark.css', resource=True) }}" type="text/css" media="(prefers-color-scheme: dark)" id="css-dark"/>
-    <link rel="stylesheet" href="{{ pathto('_static/pygments.css', resource=True) }}" type="text/css" media="(prefers-color-scheme: light)" id="pyg" />
+    <link rel="stylesheet" href="{{ pathto('_static/pygments.css', resource=True) }}" type="text/css" media="(prefers-color-scheme: light)" id="pyg-light" />
     <link rel="stylesheet" href="{{ pathto('_static/pygments_dark.css', resource=True) }}" type="text/css" media="(prefers-color-scheme: dark)" id="pyg-dark" />
     <link rel="alternate" type="application/rss+xml" title="Latest PEPs" href="https://www.python.org/dev/peps/peps.rss">
     <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
     <meta name="description" content="Python Enhancement Proposals (PEPs)"/>
 </head>
 <body>
+    {% include "partials/icons.html" %}
+    <script>
+        // set colour scheme from local storage synchronously to avoid a flash of unstyled content
```

Comment:
Make this a Jinja comment, reduces the page size ever so slightly
```suggestion
{# set colour scheme from local storage synchronously to avoid a flash of unstyled content #}
```